src/runtime/Makefile

```diff
@@ -271,6 +271,12 @@ DEFBINDMOUNTS := []
 # Image Service Offload
 DEFSERVICEOFFLOAD ?= false
 
+# SNP
+DEFSNPGUEST ?= true
```

Comment:
Potentially default this to false. In any case, we'll have to define this variable when building either vanilla/CC.

Reply:
This is only present in CCv0, not main, hence why it's true by default. For vanilla, we'll want to introduce a variable and set this when building in the spec.

Reply:
Followed up offline here, looks good, our forked code seems to need this.